script/vsts/get-release-version.js

```diff
@@ -4,8 +4,10 @@ const request = require('request-promise-native');
 const repositoryRootPath = path.resolve(__dirname, '..', '..');
 const appMetadata = require(path.join(repositoryRootPath, 'package.json'));
 
-const REPO_OWNER = process.env.REPO_OWNER;
-const NIGHTLY_RELEASE_REPO = process.env.NIGHTLY_RELEASE_REPO;
+const REPO_OWNER = process.env.REPO_OWNER ? process.env.REPO_OWNER : 'atom';
```

Comment:
Maybe this to make it more compact?
```suggestion
const REPO_OWNER = process.env.REPO_OWNER || 'atom';
```